src/vanilla/atom.ts

```diff
@@ -1,3 +1,5 @@
+import type { INTERNAL_PrdStore as Store } from './store'
```

Comment:
```suggestion
import type { PrdOrDevStore as Store } from './store'
```

Reply:
That's correct for the current implementation, but as I plan to remove DevStore by default in the future, this limits more. Maybe I did wrong with `S extends Store` too.